tools/README.md

```diff
@@ -1,6 +1,6 @@
 This directory contains command line utilities intended for use by spec editors and contributors when authoring or reviewing changes.
 
-- `reformat_js` - applies [clang-format](https://clang.llvm.org/docs/ClangFormat.html) to JavaScript blocks in the spec.
+- `reformat-js.py` - applies [clang-format](https://clang.llvm.org/docs/ClangFormat.html) to JavaScript blocks in the spec.
 - `lint.mjs` - analyses the spec Bikeshed source and generated HTML to look for common errors like duplicate words or unclosed links, and helps enforce the [coding conventions](../docs/SpecCodingConventions.md).
 
 The tools assume a POSIX-like command line environment, and have dependencies on various languages and libraries. Read the sources for more details.
```

Comment:
Nit: you may want to mention developers can execute `npm install` to install the node.js dependencies.

Reply:
Done in 1a1e054 but corrections welcome!